lmdeploy/serve/turbomind/chatbot.py

```diff
@@ -68,6 +69,7 @@ def __init__(self,
                  ignore_eos: bool = False,
                  log_level: int = logging.INFO,
                  display: bool = False,
+                 gen_cfg: GenerationConfig = GenerationConfig(),
```

Comment:
1. The initialization in https://github.com/InternLM/lmdeploy/blob/920d7196336f437e6383f39c04804236ef5270da/benchmark/profile_serving.py#L84 also should be fixed.
2. Why don't put `ignore_eos` to GenerationConfig?

Reply:
`GenerationConfig` is not supposed to be used in the init function. I just chose a quick while not elegant way to fix the bug.

Reply:
Is this parameter deprecated? Maybe we can remove it. https://github.com/InternLM/lmdeploy/blob/e7b2833441be6a4ac374ec191001ba6ae97647e0/benchmark/profile_serving.py#L86